Core/Utilities/ElasticSearchDB.py

```diff
@@ -0,0 +1,272 @@
+########################################################################
+# $Id: $
+########################################################################
+
+"""
+This class a wrapper around elasticsearch-py. It is used to query
+Elasticsearch database.
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                      import gLogger, S_OK, S_ERROR
+from elasticsearch              import Elasticsearch
+from elasticsearch_dsl          import Search, Q, A
+from elasticsearch.exceptions   import ConnectionError, TransportError, NotFoundError
+from elasticsearch.helpers      import BulkIndexError
+from elasticsearch              import helpers
+from datetime                   import datetime
+from DIRAC.Core.Utilities       import Time
```

Comment:
Well, this is yet to be discussed, but what for do you need ? Just to get epoch ? If we decide to get ride of it, it is better not to use it in new places